### PR TITLE
Remove references to martsy

### LINKED ISSUE #2308: Remove references to m.artsy.net
We should have phone Eigen point to www now that the projects are merged.

### PR DESCRIPTION
**Pull request by alloy**
Fixes #2308

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Artsy/View_Controllers/Web_Browsing/ARInternalMobileWebViewController.m [hunk 1]
```diff
@@ -156,12 +156,12 @@ - (WKNavigationActionPolicy)shouldLoadNavigationAction:(WKNavigationAction *)nav
...
             }
 
-            ARActionLog(@"Martsy URL: Denied - %@ - %@", URL, @(navigationAction.navigationType));
+            ARActionLog(@"Artsy URL: Denied - %@ - %@", URL, @(navigationAction.navigationType));
             return WKNavigationActionPolicyCancel;
         }
     }
 
-    ARActionLog(@"Martsy URL: Allowed - %@ - %@", URL, @(navigationAction.navigationType));
+    ARActionLog(@"Artsy URL: Allowed - %@ - %@", URL, @(navigationAction.navigationType));
```
Comment (orta) [inline Artsy/View_Controllers/Web_Browsing/ARInternalMobileWebViewController.m]: nice touches

File: Artsy_Tests/View_Controller_Tests/Fair/ARFairSearchViewControllerTests.m [hunk 2]
```diff
@@ -51,58 +51,51 @@ @interface Fair()
...
+//        
+//        fair.shows = [NSMutableSet set];
+//        [fairSearchVC fetchSearchResults:@"Leila"];
+//
+//        // the results are a combination of the local Leila Heller and the remote Leila Pazooki
+//        expect(fairSearchVC.searchResults.count).will.equal(2);
+//        expect(fairSearchVC.searchResults.firstObject).to.beKindOf([SearchResult class]);
+//        SearchResult *result = (id) fairSearchVC.searchResults.firstObject;
+//        expect(result.modelID).to.equal(@"leila-heller-gallery-show");
+//    });
```
Comment (orta) [inline Artsy_Tests/View_Controller_Tests/Fair/ARFairSearchViewControllerTests.m]: seems reasonable enough

**Comment by mennenia** [comment]
@alloy awesommeee! 🎉

Could you fix the changelog merge conflict? :)

**Comment by alloy** [comment]
Done. Merging!

**Pull request merged**